test/libponyc-run/runner.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+exit_code=0
+
+for directory in $(find ./ -maxdepth 1 -type d ! -iname ".*")
```

Comment:
If we only support bash then this can be rewritten in a better fashion (in fact, shellcheck complains about this). However, the "better fashion" isn't posix compat.

Reply:
this was rewritten as we decided to do bash.